client/signup/jetpack-connect/plans-grid.jsx

```diff
@@ -10,22 +11,38 @@ import Main from 'components/main';
 import StepHeader from '../step-header';
 import PlansFeaturesMain from 'my-sites/plans-features-main';
 
-export default React.createClass( {
-	displayName: 'JetpackPlansGrid',
+class JetpackPlansGrid extends Component {
+	static propTypes = {
+		basePlansPath: PropTypes.string.isRequired,
+		hideFreePlan: PropTypes.bool,
```

Comment:
@tyxla I think that's worthy of another PR, there are a few small but significant changes that can be made related to defaults. I'm going to handle that separately.